common/v2/components/WalletList/WalletList.tsx

```diff
@@ -99,7 +99,9 @@ export default class WalletList extends PureComponent<Props> {
         )}
         <WalletsContainer>
           {validWallets.map((wallet: IStory) => {
-            const walletInfo = WALLETS_CONFIG[wallet.name];
+            const walletInfo =
+              wallet.name === WalletId.WEB3 ? getWeb3Config() : WALLETS_CONFIG[wallet.name];
```

Comment:
We don't have any TypeGuards on `name` so WALLETS_CONFIG[wallet.name] may return `undefined`. Maybe use `wallet.type` and `wallet.id` here?
```
    wallet.type === WalletType.WEB3 ? getWeb3Config() : WALLETS_CONFIG[wallet.id];
```

Reply:
`wallet` is an `IStory` which by definion has the `name` property with type `WalletId`. They're defined in https://github.com/MyCryptoHQ/MyCrypto/blob/822e1b899bda559c48e8b1da72d1e5fc138bc99d/common/v2/features/AddAccount/stories.tsx

I made the type of `wallets` clearer.